gradle/libs.versions.toml

```diff
@@ -6,20 +6,23 @@ junit = "5.10.2"
 autoservice = "1.1.1"
 
 # otel SDK and API
+# updated from upstream agent with .ci/update-upstream.sh
```

Comment:
I jsut noticed that the agent bom includes the otel bom:

https://repo1.maven.org/maven2/io/opentelemetry/instrumentation/opentelemetry-instrumentation-bom/2.1.0/opentelemetry-instrumentation-bom-2.1.0.pom

We should be able to get rid of the SDK version in our projects and instead have it managed by the opentelemetry-instrumentation-bom. Could you try this?

Reply:
I think that could be a good option here, I'll try to reuse this BOM dependency so it's one less moving part for the SDK/API.